src/pages/workspace/content/review/index.tsx

```diff
@@ -26,9 +27,10 @@ import {
 import $ from "jquery";
 
 const ReviewContentSubmissions = () => {
+  const interactiveMimeTypes = ["application/vnd.ekstep.h5p-archive", "application/vnd.ekstep.html-archive"];
```

Comment:
@rajnishdargan keep this in app.config file

Reply:
Done.